_posts/2023/2023-02-15-cobbler-inside-libvirt-1.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: Cobbler inside libvirt (1)
+author: Enno
+summary: Cobbler inside libvirt with a private libvirt network
+---
+
+The last blog post that was neither a roadmap update or a release announcement was in december of 2020. Thus it is time
+to give you guys a new guide!
+
+This time we will together explore how to use Cobbler to enable a libvirt internal network to boot over the network.
+The guide will assume you have a running host OS and libvirt installed and running already. As a guest OS I will use
+openSUSE Tumbleweed but any other guest OS should work equally good.
+
+In case you have libvirt not installed please consult your OS vendors resources to do so. A few examples:
+
+- [openSUSE](https://doc.opensuse.org/documentation/leap/virtualization/html/book-virtualization/part-virt-libvirt.html)
+- [Fedora](https://developer.fedoraproject.org/tools/virtualization/installing-libvirt-and-virt-install-on-fedora-linux.html)
+- [Ubuntu](https://ubuntu.com/server/docs/virtualization-libvirt)
+
+Note:
+
+> There will be a second part of this guide where we will utilize a network bridge to also boot real hardware with our
+> VM based Cobbler setup.
+
+Information:
+
+> Depending on your setup you may need to prefix the `virsh` and other commands with `sudo`.
+
+1. Create libvirt network for Cobbler (don't use the network `default`).
```

Comment:
We may need to take a look at this wording here. As we discussed you can easily misunderstand the "don't use the network default" part.
Maybe go a little more into detail what you mean.
From what I understand it, you only need this network once you have cobbler installed? Then maybe we can move this step to after we have installed the VM to avoid confusion.